.github/actions/integration-tests/run-ci-stage1

```diff
@@ -302,6 +302,15 @@ run_cmd "crucible help"
 
 run_cmd "crucible repo info"
 
+for action in start stop; do
```

Comment:
start stop ?

Reply:
`crucible start <service>` and `crucible stop <service>`